includes/fields/class-field-post-taxonomy.php

```diff
@@ -215,25 +215,44 @@ public function tax_ajax( $post_id = null ) {
     }
 
     public function tax_select( $post_id = null ) {
-        $attr = $this->field_settings;
-
+        $attr     = $this->field_settings;
         $selected = $this->terms ? $this->terms[0] : '';
-
         $required = sprintf( 'data-required="%s" data-type="select"', $attr['required'] );
 
+        $parents      = $this->field_settings['exclude'];
+        $child_ids     = [];
+        $exclude_type = $this->exclude_type;
+
+        if ( !empty( $parents ) ) {
+            foreach ( $parents as $parent ) {
+                $terms = get_terms( 'category', array(
+                    'hide_empty' => false,
+                    'parent'     => $parent
+                ) );
+
+                foreach ( $terms as $term ) {
+                    array_push( $child_ids, $term->term_id );
+                }
+            }
+        }
+
+        if ( $this->exclude_type == 'child_of' ) {
```

Comment:
@hasinur1997 use yoda expression and tripple equal, `'child_of' === $this->exclude_type`

Reply:
ok